core/core-dotspacemacs.el

```diff
@@ -171,6 +171,9 @@ start.")
 (defvar dotspacemacs-helm-position 'bottom
   "Position in which to show the `helm' mini-buffer.")
 
+(defvar dotspacemacs-helm-use-swiper nil
```

Comment:
This can probably be a variable local to the layer, defined in a file `config.el`.